### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.7 es 4.13 (30 ottobre)

### DISCUSSION THREAD

**Discussion by erinrrr** [sapienzastudentsnetwork/calcolo-delle-probabilita#310] [Esercizi M-Z]
![image](https://github.com/user-attachments/assets/32d9770c-75b6-432e-9348-cc985aa7829b)

**Comment by erinrrr** [comment]
![image](https://github.com/user-attachments/assets/d7c7acee-0eb3-4245-a135-2ffbddbc11e4)
![image](https://github.com/user-attachments/assets/d69967d4-bc94-43f6-a495-3106cf8ad9fb)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/2fbf9357-a8b4-48f9-97f4-0592287f4476)
